src/test/java/com/puppycrawl/tools/checkstyle/internal/AllTestsTest.java

```diff
@@ -180,7 +180,9 @@ private static void verifyInputFile(Map<String, List<String>> allTests, boolean
             // moved
             final String folderPath = packge;
             slash = packge.lastIndexOf(File.separatorChar);
-            packge = path.substring(0, slash);
+            if (slash != -1) {
+                packge = path.substring(0, slash);
```

Comment:
1. `slash` -> slashIdx
2. `packge` typo, packageName or something like this is better

Reply:
Not part of my addition anymore